2023-04-02 12:17:12,794 INFO:logging in using static token
2023-04-02 12:17:13,786 INFO:Shard ID None has connected to Gateway (Session ID: b8fde7a79c5ab4b63425a39027186242).
2023-04-02 12:17:15,788 INFO:Logged in as CC4-bot#9008
2023-04-02 12:17:15,788 ERROR:Ignoring exception in on_ready
Traceback (most recent call last):
  File "C:\Users\dorin\Documents\TEACHING\CC4\discord-bot\venv\lib\site-packages\discord\client.py", line 441, in _run_event
    await coro(*args, **kwargs)
  File "C:\Users\dorin\Documents\TEACHING\CC4\discord-bot\main.py", line 39, in on_ready
    channel = client.get_guild(998234966284582942).get_channel(998234966284582945)
AttributeError: 'NoneType' object has no attribute 'get_channel'
2023-04-02 12:22:45,979 INFO:logging in using static token
2023-04-02 12:22:46,931 INFO:Shard ID None has connected to Gateway (Session ID: 8831709299eae111ae1bd5f12aa32875).
2023-04-02 12:22:48,942 INFO:Logged in as CC4-bot#9008
2023-04-02 12:29:22,667 INFO:logging in using static token
2023-04-02 12:29:23,714 INFO:Shard ID None has connected to Gateway (Session ID: 77f16e3b2e60e8e8ed1bd9f905de8b94).
2023-04-02 12:29:25,730 INFO:Logged in as CC4-bot#9008
2023-04-02 12:30:52,737 INFO:logging in using static token
2023-04-02 12:30:53,674 INFO:Shard ID None has connected to Gateway (Session ID: 0e00966181035f6365a4706d4ee04419).
2023-04-02 12:30:55,681 INFO:Logged in as CC4-bot#9008
2023-04-02 12:30:55,681 ERROR:Ignoring exception in on_ready
Traceback (most recent call last):
  File "C:\Users\dorin\Documents\TEACHING\CC4\discord-bot\venv\lib\site-packages\discord\client.py", line 441, in _run_event
    await coro(*args, **kwargs)
  File "C:\Users\dorin\Documents\TEACHING\CC4\discord-bot\main.py", line 39, in on_ready
    channel = client.get_guild(998234966284582942).get_channel(998234966284582945)
AttributeError: 'NoneType' object has no attribute 'get_channel'
2023-04-02 12:31:51,111 INFO:logging in using static token
2023-04-02 12:31:52,020 INFO:Shard ID None has connected to Gateway (Session ID: d28c0ad44d9476b5d710b1093dd02124).
2023-04-02 12:31:54,021 INFO:Logged in as CC4-bot#9008
2023-04-02 12:37:00,268 INFO:logging in using static token
2023-04-02 12:37:01,192 INFO:Shard ID None has connected to Gateway (Session ID: b5248b107e3c5ff87f959e4fe5eebcaf).
2023-04-02 12:37:03,204 INFO:Logged in as CC4-bot#9008
2023-04-02 12:41:15,150 INFO:logging in using static token
2023-04-02 12:41:16,124 INFO:Shard ID None has connected to Gateway (Session ID: 499252af5bf43579265eff3e7990d724).
2023-04-02 12:41:18,129 INFO:Logged in as CC4-bot#9008
2023-04-02 12:41:18,129 ERROR:Ignoring exception in on_ready
Traceback (most recent call last):
  File "C:\Users\dorin\Documents\TEACHING\CC4\discord-bot\venv\lib\site-packages\discord\client.py", line 441, in _run_event
    await coro(*args, **kwargs)
  File "C:\Users\dorin\Documents\TEACHING\CC4\discord-bot\main.py", line 40, in on_ready
    channel = client.get_guild(998234966284582942).get_channel(998234966284582945)
AttributeError: 'NoneType' object has no attribute 'get_channel'
2023-04-02 12:42:00,099 INFO:logging in using static token
2023-04-02 12:42:01,052 INFO:Shard ID None has connected to Gateway (Session ID: 106071cbe31408a987d3ed6b841585df).
2023-04-02 12:42:03,063 INFO:Logged in as test-cc4#1298
2023-04-02 12:42:14,717 INFO:logging in using static token
2023-04-02 12:42:15,748 INFO:Shard ID None has connected to Gateway (Session ID: 68594ee0c5fc90273988c438c862a352).
2023-04-02 12:42:17,758 INFO:Logged in as test-cc4#1298
2023-04-02 12:42:28,984 INFO:cucicov#3642: $hello
2023-04-02 12:42:29,221 INFO:test-cc4#1298: Hello stranger!
2023-04-02 12:42:36,176 INFO:cucicov#3642: $hello
2023-04-02 12:42:36,471 INFO:test-cc4#1298: Hello stranger!
2023-04-02 12:43:26,263 INFO:test-cc4#1298: 
2023-04-02 12:44:29,282 INFO:test-cc4#1298: 
2023-04-02 12:46:54,263 INFO:logging in using static token
2023-04-02 12:46:55,282 INFO:Shard ID None has connected to Gateway (Session ID: 77b8eec66e3b36e7f07c410d5a67d043).
2023-04-02 12:46:57,296 INFO:Logged in as test-cc4#1298
2023-04-02 12:47:12,158 INFO:Joined server: 1092021650058395688
2023-04-02 12:47:12,198 INFO:test-cc4#1298: 
2023-04-02 12:49:36,218 INFO:logging in using static token
2023-04-02 12:49:37,158 INFO:Shard ID None has connected to Gateway (Session ID: 7207e94e43bfdcb9e91d72c8df2e7f58).
2023-04-02 12:49:39,171 INFO:Logged in as test-cc4#1298
2023-04-02 12:50:23,266 INFO:Joined server: 1092021650058395688
2023-04-02 12:50:23,266 ERROR:Ignoring exception in on_guild_join
Traceback (most recent call last):
  File "C:\Users\dorin\Documents\TEACHING\CC4\discord-bot\venv\lib\site-packages\discord\client.py", line 441, in _run_event
    await coro(*args, **kwargs)
  File "C:\Users\dorin\Documents\TEACHING\CC4\discord-bot\main.py", line 45, in on_guild_join
    await channel.send("hey, guys!")
AttributeError: 'NoneType' object has no attribute 'send'
2023-04-02 12:50:23,330 INFO:test-cc4#1298: 
2023-04-02 12:51:48,156 INFO:logging in using static token
2023-04-02 12:51:49,150 INFO:Shard ID None has connected to Gateway (Session ID: 0c307f7725b4328da511397c7f093e58).
2023-04-02 12:51:51,172 INFO:Logged in as test-cc4#1298
2023-04-02 12:52:03,111 INFO:Joined server: 1092021650058395688
2023-04-02 12:52:03,111 ERROR:Ignoring exception in on_guild_join
Traceback (most recent call last):
  File "C:\Users\dorin\Documents\TEACHING\CC4\discord-bot\venv\lib\site-packages\discord\client.py", line 441, in _run_event
    await coro(*args, **kwargs)
  File "C:\Users\dorin\Documents\TEACHING\CC4\discord-bot\main.py", line 46, in on_guild_join
    await channel.send("hey, guys!")
AttributeError: 'NoneType' object has no attribute 'send'
2023-04-02 12:52:03,149 INFO:test-cc4#1298: 
2023-04-02 12:52:30,744 INFO:logging in using static token
2023-04-02 12:52:31,748 INFO:Shard ID None has connected to Gateway (Session ID: b826158e9944f8384f3c164cd35c54cc).
2023-04-02 12:52:33,778 INFO:Logged in as test-cc4#1298
2023-04-02 12:52:43,163 INFO:Joined server: 1092021650058395688
2023-04-02 12:52:43,164 ERROR:Ignoring exception in on_guild_join
Traceback (most recent call last):
  File "C:\Users\dorin\Documents\TEACHING\CC4\discord-bot\venv\lib\site-packages\discord\client.py", line 441, in _run_event
    await coro(*args, **kwargs)
  File "C:\Users\dorin\Documents\TEACHING\CC4\discord-bot\main.py", line 46, in on_guild_join
    await channel.send("hey, guys!")
AttributeError: 'NoneType' object has no attribute 'send'
2023-04-02 12:52:43,549 INFO:test-cc4#1298: 
2023-04-02 12:53:15,574 INFO:logging in using static token
2023-04-02 12:53:16,463 INFO:Shard ID None has connected to Gateway (Session ID: efda4da2409457611ac4cfd6947f78e6).
2023-04-02 12:53:18,483 INFO:Logged in as test-cc4#1298
2023-04-02 12:53:29,311 INFO:Joined server: 1092021650058395688
2023-04-02 12:53:29,312 ERROR:Ignoring exception in on_guild_join
Traceback (most recent call last):
  File "C:\Users\dorin\Documents\TEACHING\CC4\discord-bot\venv\lib\site-packages\discord\client.py", line 441, in _run_event
    await coro(*args, **kwargs)
  File "C:\Users\dorin\Documents\TEACHING\CC4\discord-bot\main.py", line 46, in on_guild_join
    await channel.send("hey, guys!")
AttributeError: 'CategoryChannel' object has no attribute 'send'
2023-04-02 12:53:29,373 INFO:test-cc4#1298: 
2023-04-02 12:55:09,233 INFO:logging in using static token
2023-04-02 12:55:10,195 INFO:Shard ID None has connected to Gateway (Session ID: ed528479876d72bc848f4a8b999a7153).
2023-04-02 12:55:12,205 INFO:Logged in as test-cc4#1298
2023-04-02 12:55:28,151 INFO:Joined server: 1092021650058395688
2023-04-02 12:55:28,151 ERROR:Ignoring exception in on_guild_join
Traceback (most recent call last):
  File "C:\Users\dorin\Documents\TEACHING\CC4\discord-bot\venv\lib\site-packages\discord\client.py", line 441, in _run_event
    await coro(*args, **kwargs)
  File "C:\Users\dorin\Documents\TEACHING\CC4\discord-bot\main.py", line 47, in on_guild_join
    await channel.send("hey, guys!")
AttributeError: 'CategoryChannel' object has no attribute 'send'
2023-04-02 12:55:28,192 INFO:test-cc4#1298: 
2023-04-02 12:57:49,993 INFO:logging in using static token
2023-04-02 12:57:51,002 INFO:Shard ID None has connected to Gateway (Session ID: 8ab088f8fb35e9bd5ad0b07ecd279d2f).
2023-04-02 12:57:53,020 INFO:Logged in as test-cc4#1298
2023-04-02 12:58:07,338 INFO:Joined server: 1092021650058395688
2023-04-02 12:58:07,338 ERROR:Ignoring exception in on_guild_join
Traceback (most recent call last):
  File "C:\Users\dorin\Documents\TEACHING\CC4\discord-bot\venv\lib\site-packages\discord\client.py", line 441, in _run_event
    await coro(*args, **kwargs)
  File "C:\Users\dorin\Documents\TEACHING\CC4\discord-bot\main.py", line 51, in on_guild_join
    await channel.send("hey, guys!")
AttributeError: 'CategoryChannel' object has no attribute 'send'
2023-04-02 12:58:07,378 INFO:test-cc4#1298: 
2023-04-02 12:59:04,343 INFO:logging in using static token
2023-04-02 12:59:05,330 INFO:Shard ID None has connected to Gateway (Session ID: e62111a457f4db572e7857d546dd1048).
2023-04-02 12:59:07,372 INFO:Logged in as test-cc4#1298
2023-04-02 12:59:18,136 INFO:Joined server: 1092021650058395688
2023-04-02 12:59:18,136 ERROR:Ignoring exception in on_guild_join
Traceback (most recent call last):
  File "C:\Users\dorin\Documents\TEACHING\CC4\discord-bot\venv\lib\site-packages\discord\client.py", line 441, in _run_event
    await coro(*args, **kwargs)
  File "C:\Users\dorin\Documents\TEACHING\CC4\discord-bot\main.py", line 47, in on_guild_join
    await channel.send("hey, guys!")
AttributeError: 'CategoryChannel' object has no attribute 'send'
2023-04-02 12:59:18,152 INFO:test-cc4#1298: 
2023-04-02 13:00:17,582 INFO:logging in using static token
2023-04-02 13:00:18,559 INFO:Shard ID None has connected to Gateway (Session ID: 5a4976d5577234edbdc5512182af379c).
2023-04-02 13:00:20,575 INFO:Logged in as test-cc4#1298
2023-04-02 13:00:28,195 INFO:Joined server: 1092021650058395688
2023-04-02 13:00:28,195 ERROR:Ignoring exception in on_guild_join
Traceback (most recent call last):
  File "C:\Users\dorin\Documents\TEACHING\CC4\discord-bot\venv\lib\site-packages\discord\client.py", line 441, in _run_event
    await coro(*args, **kwargs)
  File "C:\Users\dorin\Documents\TEACHING\CC4\discord-bot\main.py", line 49, in on_guild_join
    await channel.send("hey, guys!")
AttributeError: 'CategoryChannel' object has no attribute 'send'
2023-04-02 13:00:28,216 INFO:test-cc4#1298: 
2023-04-02 13:12:33,244 INFO:logging in using static token
2023-04-02 13:12:34,172 INFO:Shard ID None has connected to Gateway (Session ID: 8c5b4b91475ea433d3d52e9c0d65c4c2).
2023-04-02 13:12:36,185 INFO:Logged in as test-cc4#1298
2023-04-02 13:12:51,143 INFO:Joined server: 1092021650058395688
2023-04-02 13:12:51,143 ERROR:Ignoring exception in on_guild_join
Traceback (most recent call last):
  File "C:\Users\dorin\Documents\TEACHING\CC4\discord-bot\venv\lib\site-packages\discord\client.py", line 441, in _run_event
    await coro(*args, **kwargs)
  File "C:\Users\dorin\Documents\TEACHING\CC4\discord-bot\main.py", line 40, in on_guild_join
    channels = guild.v
AttributeError: 'Guild' object has no attribute 'v'
2023-04-02 13:12:51,177 INFO:test-cc4#1298: 
2023-04-02 13:13:02,748 INFO:logging in using static token
2023-04-02 13:13:03,741 INFO:Shard ID None has connected to Gateway (Session ID: 0a73ebb181d0765a3d2966913fad90d8).
2023-04-02 13:13:05,743 INFO:Logged in as test-cc4#1298
2023-04-02 13:13:17,345 INFO:Joined server: 1092021650058395688
2023-04-02 13:13:17,425 INFO:test-cc4#1298: 
2023-04-02 13:13:17,610 INFO:test-cc4#1298: hey, guys!
2023-04-02 13:13:17,673 ERROR:Ignoring exception in on_guild_join
Traceback (most recent call last):
  File "C:\Users\dorin\Documents\TEACHING\CC4\discord-bot\venv\lib\site-packages\discord\client.py", line 441, in _run_event
    await coro(*args, **kwargs)
  File "C:\Users\dorin\Documents\TEACHING\CC4\discord-bot\main.py", line 52, in on_guild_join
    voice = await channelVoice.connect()
AttributeError: 'TextChannel' object has no attribute 'connect'
2023-04-02 13:14:45,281 INFO:logging in using static token
2023-04-02 13:14:46,273 INFO:Shard ID None has connected to Gateway (Session ID: a57435730c6903bcd1fdc6f40f490e65).
2023-04-02 13:14:48,284 INFO:Logged in as test-cc4#1298
2023-04-02 13:14:57,357 INFO:Joined server: 1092021650058395688
2023-04-02 13:14:57,397 INFO:test-cc4#1298: 
2023-04-02 13:14:57,572 INFO:test-cc4#1298: hey, guys!
2023-04-02 13:14:57,613 INFO:Connecting to voice...
2023-04-02 13:14:57,613 INFO:Starting voice handshake... (connection attempt 1)
2023-04-02 13:14:57,881 INFO:Voice handshake complete. Endpoint found bucharest4993.discord.media
